### ISSUE TITLE
Collapsible table of contents

### ISSUE THREAD

**Issue by christianp** [chirun-ncl/chirun#276]
The table of contents should be a tree view. 

See the Numbas explore mode breadcrumbs in the new theme for an implementation of an accessible tree view.

The branch `collapsing-toc` has a first go at an implementation, using hidden checkboxes.
